app/services/audit/config.rb

```diff
@@ -0,0 +1,6 @@
+module Audit
+  class Config
```

Comment:
this is probably not needed at all, see my other two comments about the two constants